e2e/tests/upgrades/upgrade_test.go

```diff
@@ -12,14 +12,15 @@ import (
 	"github.com/strangelove-ventures/ibctest/ibc"
 	"github.com/strangelove-ventures/ibctest/test"
 	"github.com/stretchr/testify/suite"
+	"golang.org/x/sync/errgroup"
 
 	"github.com/cosmos/ibc-go/e2e/testconfig"
 	"github.com/cosmos/ibc-go/e2e/testsuite"
 	"github.com/cosmos/ibc-go/e2e/testvalues"
 )
 
 const (
-	haltHeight         = uint64(50)
+	haltHeight         = uint64(100)
```

Comment:
the original height ended up being in the past by the time the transfers are complete